types/webidl2/index.d.ts

```diff
@@ -105,7 +188,7 @@ export interface AbstractTypeDescription extends AbstractBase {
 
 interface AbstractNonUnionTypeDescription extends AbstractTypeDescription {
     /** String indicating the generic type (e.g. "Promise", "sequence"). The empty string otherwise. */
-    generic: IDLTypeDescription["generic"];
+    generic: IDLTypeDescription['generic'];
```

Comment:
I'm okay with this change, but I'd suggest this kind of change should be in a separate PR at least. This made the PR big enough to scare me and that delayed the review.

Reply:
Sorry about that, that was Prettier kicking in and formatting the file automatically as my editor is configured to do!